source/workers/LocalWorker.cpp

```diff
@@ -57,6 +64,13 @@
 #define NETBENCH_RECEIVE_TIMEOUT_SEC	20 // max time to wait for incoming data on client & server
 #define NETBENCH_SHORT_POLL_TIMEOUT_SEC	2  // time to check for interrupts in longer poll wait loops
 
+#define TAG_CHECKSUM_LEN        2
+#define TAG_KEY_SHORT_NAME      "short"
+#define TAG_KEY_MEDIUM_NAME    "medium"
```

Comment:
let's align the words "medium", "long" and the other values underneath to all start in the same column, just so that it looks nicer :-)